LICENSE.txt

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License for the software signac-docs.
```

Comment:
Does BSD3 make sense for documentation? I know this repository also includes code so overall BSD3 is probably good, but maybe we should license documentation text differently?

Reply:
From a brief bit of googling, this is fine to choose BSD-3: https://choosealicense.com/non-software/#documentation

Reply:
My default choice for docs would have been CC BY or similar, but I think BSD is fine and removes a source of confusion between different licenses.